### DISCUSSION TITLE
Plugin for banners, interactive forms, CustomerActionEvent &CustomerReminder

### DISCUSSION THREAD

**Discussion by Nikhil13x** [grandnode/grandnode2#328] [General]
Hello, how can we contribute for these removed modules to be added as plugins? A separate repo or in the same solution as a new project?

**Comment by Nikhil13x** [comment]
@KrzysztofPajak  Any decisions made on this? We would love to contriute.

**Comment by KrzysztofPajak** [comment]
@Nikhil13x Thanks, I've send you request to https://github.com/grandnode/plugin-marketing
